content/developer/reference/backend/testing.rst

```diff
@@ -819,35 +840,39 @@ driven explicitly by the user) it's more complicated when running
 "test" tours, or when running tours through the test suite. In that
 case there are two main tricks:
 
-- Have a step with a ``run() { debugger; }`` action.
+- Have a step property ``break: true,`` in debug mode (debug=True).
+
+  This adds a debugger at the start of the step and then allows you
+  to put breakpoints in the desired locations.
+
+**Advantages**
+  - very simple
+  - the tour restarts as soon as you resume execution
+**Drawbacks**
+  - page interaction is limited as all javascript is blocked
+  - Have a step property ``pause: true,`` in debug mode (debug=True).
```

Comment:
This is one of the 2 tricks, the indentation is wrong.
Also, the "Have" again can be removed.

Reply:
Note: I saw later that all the tricks seemed weirdly indented, so maybe it's github. And it's correct when compiled as html